src/app/ubs-admin/components/shared/components/pagination/pagination.component.html

```diff
@@ -0,0 +1,18 @@
+<pagination-template #p="paginationApi" [id]="paginationId" (pageChange)="changePage($event)">
+  <div class="custom-pagination">
+    <div *ngIf="currentPage !== 1" [class.disabled]="p.isFirstPage()">
+      <a (click)="p.previous()"><mat-icon class="arrow">keyboard_arrow_left</mat-icon></a>
+    </div>
+    <div *ngFor="let page of p.pages" [class.current]="p.getCurrent() === page.value">
```

Comment:
what is p here ?

Reply:
It's from ngx-pagination. Vlad took the example of comment-paginator.component